sections/02_intro.md

```diff
@@ -6,7 +6,7 @@ social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We foresee a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
+also new challenges. We expect a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
```

Comment:
Can you use this language here? Right now your changes hit a sentence that @gwaygenomics also touched. I suspect your merge will wipe out his:
> We expect that achieving transformational advances will require algorithms that automatically extract meaningful patterns while providing actionable insights that enable us to better treat, study, and categorize disease, all within data constrained and privacy critical environments.